lms/envs/common.py

```diff
@@ -991,7 +991,7 @@
 HELP_MODAL_LINKS = []
 
 ##### Zendesk #####
-ZENDESK_URL = None
+ZENDESK_URL = 'https://stanfordonline.zendesk.com/hc/en-us'
```

Comment:
Move this to test.py, but change it to a bogus URL: `https://example.com/zendesk`